src/Elastic.Markdown/Slices/_Layout.cshtml

```diff
@@ -19,7 +19,14 @@
 }
 @(await RenderPartialAsync(_Header.Create(Model)))
 <div id="main-container" class="flex flex-col items-center border-t-1 border-grey-20">
+	<div id="dismissible-banner" class="container admonition tip hidden">
+		<p>
+			You are viewing the new Elastic Docs experience for the latest product versions. Looking for older docs? Click <strong>View previous version</strong>. 
```

Comment:
Since I don't think all pages necessarily have a previous version, I think this ought to be updated to avoid confusion:
```suggestion
			You are viewing the new Elastic Docs experience for the latest product versions. If the page exists in older docs, you can click <strong>View previous version</strong>. 
```